falcon/app.py

```diff
@@ -863,3 +864,17 @@ def _get_body(self, resp, wsgi_file_wrapper=None):
             return iterable, None
 
         return [], 0
+
+
+# NOTE(mikeyusko): This function is temporary
```

Comment:
nitpick: kinda funky line breaks in the comment here. Not a big deal at all, you might want to shoot for about an 80 character line. 🤷‍